bundle/introducing_config/README.md

```diff
@@ -0,0 +1,158 @@
+# An example of how to use the bundle config
+
+```bash
+python -m monai.bundle run display --config_file example.yaml
+```
+
+By running the command within the same directory of this readme file, an example MedNIST dataset (~60MB) will be downloaded
+and a PyTorch-style `Dataset` will be created. The yaml config file also specifies the necessary commands to display the first pair of images in the dataset (shown in the figure).
+
+<div> <img src="../../figures/mednist_config_intro.png"/> </div>
+
+More specifically:
+- `python -m monai.bundle run` is the command [provided by the `monai.bundle` module](https://docs.monai.io/en/stable/bundle.html#monai.bundle.run). Apart from `run`, there are other actions available, such as `download` and `init_bundle`.
+- `display` is a user-defined component name in the `example.yaml` config file.
+- `--config_file example.yaml` specifies the config file describing the workflow, `monai.bundle` module supports both yaml and json formats ([syntax quick ref.](https://docs.monai.io/en/stable/config_syntax.html)).
+
+The rest of the readme file will explain how the yaml config file is parsed in detail.
+
+_Pre-requisites_
+
+Installing MONAI (<https://docs.monai.io/en/stable/installation.html>) and basic knowledge of Python and Pytorch.
+
+## Parsing the `example.yaml`
+
+### Import modules
+
+The bundle parser will first evaluate all the import statements such as `import package` and `from package import mod`.
+
+```yaml
+imports:
+  - $import glob
+  - $import matplotlib.pyplot as plt
+```
+
+These two statements will be evaluated and the imported modules will be available throughout the config file.
+By default, the parser imports `monai`'s modules and `import numpy as np` for you, so you don't need to add them in the config file.
+
+### Running the `display` component
```

Comment:
Hi @wyli, maybe we can put this one after L110 to make it more consistent with the equivalent Python code.